apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx

```diff
@@ -182,14 +181,16 @@ const MapView = ({
                 {selectedRegionKey === region.region && (
                   <circle
                     r={4}
-                    className={`animate-ping ${hasNoDatabases ? 'fill-white/30' : 'fill-brand'}`}
+                    className={`animate-ping ${
+                      hasNoDatabases ? 'fill-black/30 dark:fill-white/30' : 'fill-brand'
```

Comment:
probably need `bg-alternative/30`

Reply:
sorry, `fill-background-alternative/30`

Reply:
opted for `fill-border-stronger`, `fill-background-alternative` is a little too dark on dark mode